main/src/main/java/cgeo/geocaching/connector/trackable/GeokretyConnector.java

```diff
@@ -296,12 +299,30 @@ public String getTrackableTrackingCodeFromUrl(@NonNull final String url) {
     @Nullable
     @WorkerThread
     private static String getGeocodeFromTrackingCode(final String trackingCode) {
-        final String response = Network.getResponseData(Network.getRequest(URLPROXY + "/nr2id/" + trackingCode));
+        //https://staging.geokrety.org/api/v1/export2?tracking_code={trackingCode}
+        //Example response:
+        //<gkxml version="1.0" date="2024-08-07 19:02:22" date_Iso8601="2024-08-07T19:02:22+00:00">
+        //<geokrety>
+        //<geokret id="66184" type="1" collectible="false" owner_id="26422" ownername="kumy" holder_id="26422" holdername="" dist="36291" date="2017-12-13" places="12" image="GK10288_6503b68749e8a">
+        //<![CDATA[ Super Picsou Geant #200 ]]>
+        //</geokret>
+        //</geokrety>
+        //</gkxml>
+
+        final String response = Network.getResponseData(Network.getRequest("https://staging.geokrety.org/api/v1/export2?tracking_code=" + trackingCode));
```

Comment:
SUing the staging env is not a good idea as it's not prod !

And I thought there was some variable for the url: `URLPROXY`

Reply:
Changed to URL (which points to https://geokrety.org). I also tried URLPROXY (points to https://api.geokrety.org), but that gave me HTTP404

Reply:
I don't remember what happen to https://api.geokrety.org/ during the transition GKv2. Maybe it has been abandoned. Need to check

Reply:
The `api.geokrety.org` url is not abandoned, it does only redirects. Some sample usages:
- `curl https://api.geokrety.org/gk/GKb65c/ -L`
- `curl https://api.geokrety.org/gk/GKb65c/details/ -L`
- `curl https://api.geokrety.org/wpt/GC5brqk/ -L`
- `curl https://api.geokrety.org/wpt/GC5brqk/details/ -L`

However it does not contain endpoint for searching by TC

You can check the nginx config here https://github.com/geokrety/geokrety-nginx/blob/main/files/etc/nginx/templates/default.conf.template#L183-L280

Reply:
I remember now, this api is a relic of the old GeoKretyMap Api where I didn't had access to TC.
Better not using it anyway...